### ISSUE TITLE
Null pointer exception related to catnav.cfc when upgrading to 7.4.0

### ISSUE THREAD

**Issue by no3rdw** [MasaCMS/MasaCMS#178]
I'm not sure if I should be posting this to Issues or Discussions, so I'm sorry if this is the wrong place!

I just upgraded a large multi-site Masa install running 7.3.6 to 7.4.0 via the 'Update Masa CMS Core' button in the interface and was met with a Null Pointer Exception after refreshing the application.  The error message was helpful in tracking down the source of the problem - it seems to be related to when the [catnav.cfc](https://github.com/MasaCMS/MasaCMS/blob/main/core/modules/v1/nav/model/beans/catnav.cfc) component is registered, but I'm not sure _why_ this file is causing an issue, especially when it hasn't been modified in 8 months.

Here is what I know:  

- Catnav.cfc was not present in my 7.3.6 install
- When catnav.cfc exists in my core directory, I get the Null Pointer Exception below immediately after refreshing the application, both for the Masa admin panel and when accessing the website itself.
- When I delete catnav.cfc from the core and refresh the application, all seems to function properly (in the admin and on the live site), including the display of categories (although my site template only uses the Category Summary component, not Category Navigation)
- This site was previously migrated to Masa 7.3.X from Mura 7, and previous to that was migrated from Mura 6 - there may be some weirdness in the site DB related to categories that is causing this issue.  (Let me know if you would like me to provide specific tcontentcategories data to help debug this further)
- I'm currently running ColdFusion (2018,0,13,329786)

The null pointer exception error is below - I can follow that it's registering the catcav component object, gets to line 1951 in configBean.cfc where it is getting metadata on the component's path... 
```
Server Error

500 - Internal server error.
There is a problem with the resource you are looking for, and it cannot be displayed.
500 Error
The CFML compiler encountered an unexpected java.lang.NullPointerException exception.
Type:Template
The reason for this was: '' Occurred at:<pre>java.lang.NullPointerException at coldfusion.compiler.ASTvariableDefinition.setVariableName(ASTvariableDefinition.java:43) at coldfusion.compiler.cfml40.localVariableDefinition(cfml40.java:5557) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1916) at coldfusion.compiler.cfml40.cfscriptBlock(cfml40.java:3231) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1992) at coldfusion.compiler.cfml40.ComplexStatement(cfml40.java:2140) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1978) at coldfusion.compiler.cfml40.consumeFunctionBody(cfml40.java:4320) at coldfusion.compiler.cfml40.functionDefinition(cfml40.java:4255) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1997) at coldfusion.compiler.cfml40.cfscript(cfml40.java:1781) at coldfusion.compiler.cfml40.processCfmlTags(cfml40.java:6563) at coldfusion.compiler.cfml40.cfml(cfml40.java:6609) at coldfusion.compiler.cfml40.start(cfml40.java:7070) at coldfusion.compiler.NeoTranslator.parsePag...</pre>
coldfusion.compiler.ParseException$CompilerExceptionWrapper: The CFML compiler encountered an unexpected java.lang.NullPointerException exception.
	at coldfusion.compiler.ParseException.wrap(ParseException.java:385)
	at coldfusion.compiler.cfml40.start(cfml40.java:7212)
	at coldfusion.compiler.NeoTranslator.parsePage(NeoTranslator.java:655)
	at coldfusion.compiler.NeoTranslator.parsePage(NeoTranslator.java:635)
	at coldfusion.compiler.NeoTranslator.parseAndTransform(NeoTranslator.java:457)
	at coldfusion.compiler.NeoTranslator.translateJava(NeoTranslator.java:407)
	at coldfusion.compiler.NeoTranslator.translateJava(NeoTranslator.java:160)
	at coldfusion.runtime.TemplateClassLoader$TemplateCache$1.fetch(TemplateClassLoader.java:478)
	at coldfusion.util.LruCache.get(LruCache.java:180)
	at coldfusion.runtime.TemplateClassLoader$TemplateCache.fetchSerial(TemplateClassLoader.java:404)
	at coldfusion.util.AbstractCache.fetch(AbstractCache.java:58)
	at coldfusion.util.SoftCache.get_statsOff(SoftCache.java:153)
	at coldfusion.util.SoftCache.get(SoftCache.java:92)
	at coldfusion.runtime.TemplateClassLoader.findClass(TemplateClassLoader.java:672)
	at coldfusion.runtime.RuntimeServiceImpl.getFile(RuntimeServiceImpl.java:1180)
	at coldfusion.runtime.RuntimeServiceImpl.resolveTemplatePath(RuntimeServiceImpl.java:1132)
	at coldfusion.runtime.TemplateProxyFactory.getResolvedFile(TemplateProxyFactory.java:1409)
	at coldfusion.runtime.TemplateProxyFactory.getTemplateFileHelper(TemplateProxyFactory.java:1753)
	at coldfusion.cfc.ComponentProxyFactory.getProxy(ComponentProxyFactory.java:77)
	at coldfusion.cfc.ComponentProxyFactory.getProxy(ComponentProxyFactory.java:56)
	at coldfusion.runtime.CFPage.CreateObject(CFPage.java:8344)
	at coldfusion.runtime.CFPage.CreateObject(CFPage.java:8363)
	at cfconfigBean2ecfc1000861744$funcREGISTERBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1951)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:609)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4116)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4097)
	at cfconfigBean2ecfc1000861744$funcREGISTERBEANDIR.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1912)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:609)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4116)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4097)
	at cfconfigBean2ecfc1000861744$funcREGISTERBEANDIR.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1910)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:609)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:835)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:641)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3549)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3524)
	at cfsettingsManager2ecfc1017686926$funcSETSITES.runFunction(D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc:499)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4175)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4155)
	at cfsettingsManager2ecfc1017686926$funcINIT.runFunction(D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc:90)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:609)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:835)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:641)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3549)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3524)
	at Statement0.evaluate(<generated>:1)
	at coldfusion.compiler.ExprClassLoader.evaluate(ExprClassLoader.java:149)
	at coldfusion.runtime.CFPage.Evaluate(CFPage.java:3487)
	at cfioc2ecfc363062062$funcRESOLVEBEANCREATE.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:740)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.runtime.UDFMethod$ReturnTypeFilter.invoke(UDFMethod.java:485)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4175)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4155)
	at cfioc2ecfc363062062$funcRESOLVEBEANCREATE.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:717)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.runtime.UDFMethod$ReturnTypeFilter.invoke(UDFMethod.java:485)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4175)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4155)
	at cfioc2ecfc363062062$funcRESOLVEBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:620)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.runtime.UDFMethod$ReturnTypeFilter.invoke(UDFMethod.java:485)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4175)
	at coldfusion.runtime.CfJspPage._invokeUDF(CfJspPage.java:4155)
	at cfioc2ecfc363062062$funcGETBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:132)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.runtime.UDFMethod$ReturnTypeFilter.invoke(UDFMethod.java:485)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:830)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:613)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:438)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3627)
	at coldfusion.runtime.CfJspPage._invoke(CfJspPage.java:3604)
	at cfonApplicationStart_include2ecfm525585626._factor36(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:494)
	at cfonApplicationStart_include2ecfm525585626._factor48(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:129)
	at cfonApplicationStart_include2ecfm525585626.runPage(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:1)
	at coldfusion.runtime.CfJspPage.invoke(CfJspPage.java:262)
	at coldfusion.tagext.lang.IncludeTag.handlePageInvoke(IncludeTag.java:735)
	at coldfusion.tagext.lang.IncludeTag.doStartTag(IncludeTag.java:565)
	at coldfusion.runtime.CfJspPage._emptyTcfTag(CfJspPage.java:4329)
	at cfonRequestStart_include2ecfm1304416642._factor0(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:131)
	at cfonRequestStart_include2ecfm1304416642._factor4(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:114)
	at cfonRequestStart_include2ecfm1304416642._factor7(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:2)
	at cfonRequestStart_include2ecfm1304416642.runPage(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:1)
	at coldfusion.runtime.CfJspPage.invoke(CfJspPage.java:262)
	at coldfusion.tagext.lang.IncludeTag.handlePageInvoke(IncludeTag.java:735)
	at coldfusion.tagext.lang.IncludeTag.doStartTag(IncludeTag.java:565)
	at coldfusion.runtime.CfJspPage._emptyTcfTag(CfJspPage.java:4329)
	at cfApplication2ecfc1429852049$funcONREQUESTSTART.runFunction(D:\inetpub\wwwroot\installdir\admin\Application.cfc:209)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:554)
	at coldfusion.filter.SilentFilter.invoke(SilentFilter.java:47)
	at coldfusion.runtime.UDFMethod$ArgumentCollectionFilter.invoke(UDFMethod.java:448)
	at coldfusion.filter.FunctionAccessFilter.invoke(FunctionAccessFilter.java:95)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:399)
	at coldfusion.runtime.UDFMethod.runFilterChain(UDFMethod.java:372)
	at coldfusion.runtime.UDFMethod.invoke(UDFMethod.java:288)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:830)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:613)
	at coldfusion.runtime.TemplateProxy.invoke(TemplateProxy.java:438)
	at coldfusion.runtime.AppEventInvoker.invoke(AppEventInvoker.java:115)
	at coldfusion.runtime.AppEventInvoker.onRequestStart(AppEventInvoker.java:286)
	at coldfusion.filter.ApplicationFilter.invoke(ApplicationFilter.java:544)
	at coldfusion.filter.RequestMonitorFilter.invoke(RequestMonitorFilter.java:43)
	at coldfusion.filter.MonitoringFilter.invoke(MonitoringFilter.java:40)
	at coldfusion.filter.PathFilter.invoke(PathFilter.java:162)
	at coldfusion.filter.IpFilter.invoke(IpFilter.java:45)
	at coldfusion.filter.ExceptionFilter.invoke(ExceptionFilter.java:96)
	at coldfusion.filter.ClientScopePersistenceFilter.invoke(ClientScopePersistenceFilter.java:28)
	at coldfusion.filter.BrowserFilter.invoke(BrowserFilter.java:38)
	at coldfusion.filter.NoCacheFilter.invoke(NoCacheFilter.java:60)
	at coldfusion.filter.GlobalsFilter.invoke(GlobalsFilter.java:38)
	at coldfusion.filter.DatasourceFilter.invoke(DatasourceFilter.java:22)
	at coldfusion.filter.CachingFilter.invoke(CachingFilter.java:62)
	at coldfusion.CfmServlet.service(CfmServlet.java:226)
	at coldfusion.bootstrap.BootstrapServlet.service(BootstrapServlet.java:311)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:228)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at coldfusion.monitor.event.MonitoringServletFilter.doFilter(MonitoringServletFilter.java:46)
	at coldfusion.bootstrap.BootstrapFilter.doFilter(BootstrapFilter.java:47)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:373)
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:462)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1723)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
	at coldfusion.compiler.ASTvariableDefinition.setVariableName(ASTvariableDefinition.java:43)
	at coldfusion.compiler.cfml40.localVariableDefinition(cfml40.java:5557)
	at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1916)
	at coldfusion.compiler.cfml40.cfscriptBlock(cfml40.java:3231)
	at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1992)
	at coldfusion.compiler.cfml40.ComplexStatement(cfml40.java:2140)
	at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1978)
	at coldfusion.compiler.cfml40.consumeFunctionBody(cfml40.java:4320)
	at coldfusion.compiler.cfml40.functionDefinition(cfml40.java:4255)
	at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:1997)
	at coldfusion.compiler.cfml40.cfscript(cfml40.java:1781)
	at coldfusion.compiler.cfml40.processCfmlTags(cfml40.java:6563)
	at coldfusion.compiler.cfml40.cfml(cfml40.java:6609)
	at coldfusion.compiler.cfml40.start(cfml40.java:7070)
	... 187 more

Column: 65
Line: 18
Template: D:\inetpub\wwwroot\installdir\core\modules\v1\nav\model\beans\catnav.cfc
Type: SYNTAX

Column: 0
ID: CF_CFPAGE
Line: 1951
Raw Trace: at cfconfigBean2ecfc1000861744$funcREGISTERBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1951)
Template: D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 1912
Raw Trace: at cfconfigBean2ecfc1000861744$funcREGISTERBEANDIR.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1912)
Template: D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 1910
Raw Trace: at cfconfigBean2ecfc1000861744$funcREGISTERBEANDIR.runFunction(D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc:1910)
Template: D:\inetpub\wwwroot\installdir\core\mura\configBean.cfc
Type: CFML

Column: 0
ID: CF_TEMPLATEPROXY
Line: 499
Raw Trace: at cfsettingsManager2ecfc1017686926$funcSETSITES.runFunction(D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc:499)
Template: D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 90
Raw Trace: at cfsettingsManager2ecfc1017686926$funcINIT.runFunction(D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc:90)
Template: D:\inetpub\wwwroot\installdir\core\mura\settings\settingsManager.cfc
Type: CFML

Column: 0
ID: CF_TEMPLATEPROXY
Line: 1
Raw Trace: at Statement0.evaluate(<generated>:1)
Template: <generated>
Type: CFML

Column: 0
ID: CF_CFPAGE
Line: 740
Raw Trace: at cfioc2ecfc363062062$funcRESOLVEBEANCREATE.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:740)
Template: D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 717
Raw Trace: at cfioc2ecfc363062062$funcRESOLVEBEANCREATE.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:717)
Template: D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 620
Raw Trace: at cfioc2ecfc363062062$funcRESOLVEBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:620)
Template: D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc
Type: CFML

Column: 0
ID: CF_UDFMETHOD
Line: 132
Raw Trace: at cfioc2ecfc363062062$funcGETBEAN.runFunction(D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc:132)
Template: D:\inetpub\wwwroot\installdir\core\mura\bean\ioc.cfc
Type: CFML

Column: 0
ID: CF_TEMPLATEPROXY
Line: 494
Raw Trace: at cfonApplicationStart_include2ecfm525585626._factor36(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:494)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm
Type: CFML

Column: 0
ID: CF_ONAPPLICATIONSTART_INCLUDE
Line: 129
Raw Trace: at cfonApplicationStart_include2ecfm525585626._factor48(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:129)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm
Type: CFML

Column: 0
ID: CF_ONAPPLICATIONSTART_INCLUDE
Line: 1
Raw Trace: at cfonApplicationStart_include2ecfm525585626.runPage(D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm:1)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onApplicationStart_include.cfm
Type: CFML

Column: 0
ID: CFINCLUDE
Line: 131
Raw Trace: at cfonRequestStart_include2ecfm1304416642._factor0(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:131)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm
Type: CFML

Column: 0
ID: CF_ONREQUESTSTART_INCLUDE
Line: 114
Raw Trace: at cfonRequestStart_include2ecfm1304416642._factor4(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:114)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm
Type: CFML

Column: 0
ID: CF_ONREQUESTSTART_INCLUDE
Line: 2
Raw Trace: at cfonRequestStart_include2ecfm1304416642._factor7(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:2)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm
Type: CFML

Column: 0
ID: CF_ONREQUESTSTART_INCLUDE
Line: 1
Raw Trace: at cfonRequestStart_include2ecfm1304416642.runPage(D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm:1)
Template: D:\inetpub\wwwroot\installdir\core\appcfc\onRequestStart_include.cfm
Type: CFML

Column: 0
ID: CFINCLUDE
Line: 209
Raw Trace: at cfApplication2ecfc1429852049$funcONREQUESTSTART.runFunction(D:\inetpub\wwwroot\installdir\admin\Application.cfc:209)
Template: D:\inetpub\wwwroot\installdir\admin\Application.cfc
Type: CFML

```

**Comment by no3rdw** [comment]
Just as an update to this, I installed a brand new install of Masa 7.4.0 using the installer and a blank datasource, and this Null Pointer Error is displayed immediately upon trying to login to the Masa admin for the first time.  Deleting the catnav.cfc (or renaming it to a different extension) immediately resolves the problem.

My server details displayed in Masa are:
App Server ColdFusion (2018,0,13,329786)
DB Server Microsoft SQL Server (Microsoft SQL Server 2017 - 14.0.3192.2)
OS Version Windows Server 2016 (10.0)

**Comment by guustnieuwenhuis** [comment]
fix will be part of the 7.4.3 patch release
